market/demos/master-detail/meta.json

```diff
@@ -1,7 +1,7 @@
 {
 	"id": "master-detail-demo",
-	"name": "Master Detail",
-	"description": "A utility process with one master dialog(with detail view) and a stand-alone detail dialog to implement a master detail scenario with Ivy and Primefaces.",
+	"name": "Master-Detail interface",
+	"description": "Exemplary UI possibilities in Axon Ivy: Utilizing the Master-Detail pattern as a best practice.",
```

Comment:
looks as if we didn't use the latest claim @nqhoan-axonivy  ... maybe next time we better wait until we have an explicit go from the business. if we get no response within 24h hours ... we will just merge as is and correct later on request.